packages/tests/tests/peer-exchange/query.spec.ts

```diff
@@ -123,7 +123,8 @@ describe("Peer Exchange Query", function () {
     await tearDownNodes([nwaku1, nwaku2, nwaku3], waku);
   });
 
-  it("connected peers and dial", async function () {
+  // slow and flaky in CI
```

Comment:
let's open an issue to track this please

Reply:
Thanks, I've opened [this](https://github.com/waku-org/js-waku/issues/1899) one